SR2019/2019-03-11-competition-information.md

```diff
@@ -0,0 +1,57 @@
+---
+to: Student Robotics 2019
+subject: Competition Informatiom
+---
+
+Hi all,
+
+There's just 26 days between now and the competition, and we're getting the final things ready for Student Robotics 2019. We hope your robots are competition-ready, and you're well on your way to entering a robot capable of winning!
+
+We have some final information for you for the competition.
+
+## Venue Layout
+
+The team pits for this year's competition are spread over 3 floors. If any members of your team have reduced mobility, or any other specific requirements which may impact the location of your team pit, please let us know as soon as possible so we can account for this when placing teams.
+
+## Media Consent Form and Tickets
+
+As with previous years, we require you to sign a media consent form. These media consent forms allow us to use videos and photographs taken at the event for future marketing purposes.
+
+[Download our Media Consent form](https://studentrobotics.org/resources/sr2019/media-consent.pdf)
+
+If a member of your team is under the age of 16, they will need to get a parent or guardian to sign on their behalf. Please put "Student Robotics 2019" as the event name.
+
+Once these forms have been completed, please send them to `teams@studentrobotics.org` with the word 'media-consent' in the subject line, to ensure they won't be missed. We will have media consent forms available at the event, but pre-processing them makes the reception process faster.
+
+Once your form has been processed, you'll receive an automated email response with your ticket. These tickets must be brought with you to the competition (printed or on a phone is fine).
```

Comment:
Does scanning from a phone definitely work?

Reply:
I'm about 90% sure _I_ got in to an event using the ticket on my phone, so I believe that's enough. I'll give it another test at some point to check, but I think we can ship this regardless. Worst comes to worst we confirm it's a valid ticket visually, and manually enter the username to check someone in.

Reply:
I'm not sure manually entering the name is equivalent - the tickets are cryptographically signed.